packages/core/lib/logger.js

```diff
@@ -38,7 +36,19 @@ function calculateLogLevel(level) {
 }
 
 function createTimestamp() {
-  return format(new Date(), 'YYYY-MM-DD HH:mm:ss.SSS Z');
+  var now = new Date(),
+      tzo = -now.getTimezoneOffset(),  // Negate to make this tzo = local - UTC
+      dif = tzo >= 0 ? '+' : '-',
+      pad = function(num) {
+          var norm = Math.floor(Math.abs(num));
+          return (norm < 10 ? '0' : '') + norm;
+      };
+
+  return new Date(now.getTime() + (tzo * 60 * 1000)).toISOString()
+    .replace(/T/, ' ')
```

Comment:
Shall we use 4 space indent for all lines?

Reply:
Good call.